specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4003,6 +4003,215 @@
           }
         }
       }
+    },
+    "/{scope}/providers/Microsoft.Resources/tags/default": {
+      "put": {
+        "tags": [
+          "Tags"
+        ],
+        "operationId": "Tags_ResourceCreate",
+        "description": "Create or Replace existing tags with pass-in tags.",
+        "parameters": [
+          {
```

Comment:
Reference the shared "ScopeParameter" definition (might need to make the description more generic)

Reply:
Referenced "ScopeParameter" in parameters, and changed the description to "The resource scope".